src/RokuDeploy.spec.ts

```diff
@@ -684,9 +723,56 @@ describe('index', () => {
             expect(err.message.startsWith('Cannot zip'), `Unexpected error message: "${err.message}"`).to.be.true;
         });
 
+        it('should create zip at zipPath location', async () => {
+            const testZipPath = path.join(tempDir, 'custom-location', 'my-app.zip');
+            options.zipPath = testZipPath;
+            options.stagingDir = stagingDir;
+
+            // Create staging directory with manifest
+            fsExtra.ensureDirSync(stagingDir);
+            fsExtra.writeFileSync(path.join(stagingDir, 'manifest'), '');
+
+            await rokuDeploy.zipPackage(options);
+
+            // Check that zip was created at zipPath location
+            expectPathExists(testZipPath);
+
+            // Check that it wasn't created at the default location
+            const defaultPath = path.join(options.outDir, options.outFile + '.zip');
+            expect(fsExtra.existsSync(defaultPath)).to.be.false;
+        });
+
+        it('should create zip at relative zipPath location', async () => {
+            const testZipPath = './custom-relative/my-app.zip';
+            options.zipPath = testZipPath;
+            options.stagingDir = stagingDir;
+
+            // Create staging directory with manifest
+            fsExtra.ensureDirSync(stagingDir);
+            fsExtra.writeFileSync(path.join(stagingDir, 'manifest'), '');
+
+            await rokuDeploy.zipPackage(options);
+
+            // Check that zip was created at resolved relative path
+            expectPathExists(path.resolve(testZipPath));
+        });
+
     });
 
     describe('createPackage', () => {
+        it('works with zipPath option', async () => {
```

Comment:
`createPackage()`, along with a few other commands like `deployAndSignPackage()` and `deploy()`, were moved to the CLI commands in the latest branch `v4`. Please look at that branch and the branch `new-args-for-commands` for the latest updates. I've pointed this PR to the branch `new-args-for-commands` since that is what all these changes should be based off of.